Documentation/contributing.rst

```diff
@@ -210,9 +210,22 @@ commands, respectively:
     $ service cilium status
     $ cilium status
 
+.. _testsuite:
+
 Testsuite
 ~~~~~~~~~
 
+.. Warning::
+
+  Running the testsuite will modify the host environment. If you are using the
+  default VM that might not be an issue, but if you are running bare-metal or a
+  different VM the tests might fail or worst case remove possibly important
+  configuration. Specifically, they are modifying tables and flushing chains
+  via ``iptables``, kernel configurations change for IP via ``sysctl``. Also
```

Comment:
Looks great, this one sentence should be reworded to:

    Specifically, they are modifying the state and configuration of system including,
    changes to ``iptables`` configuration, kernel configuration via ``sysctl``, adding
    and removing networking devices, routes, etc. via ``iproute2``.